src/components/Link/tests/Link.test.tsx

```diff
@@ -123,4 +123,32 @@ describe('<Link />', () => {
       });
     });
   });
+
+  describe('accessibilityLabel', () => {
+    it('passes prop to the button url is not provided', () => {
+      const mockAccessibilityLabel = 'mock accessibility label';
+      const link = mountWithApp(
+        <Link accessibilityLabel={mockAccessibilityLabel} />,
+      );
+
+      expect(link).toContainReactComponent('button', {
+        'aria-label': mockAccessibilityLabel,
+      });
+    });
+
+    it('passes prop', () => {
+      const mockAccessibilityLabel = 'mock accessibility label';
+
+      const link = mountWithApp<LinkProps>(
+        <Link
+          url="https://shopify.com"
+          accessibilityLabel={mockAccessibilityLabel}
+        />,
+      );
+
+      expect(link).toContainReactComponent('a', {
```

Comment:
My bad. I just left an edit saying I think this needs to be calling `UnstyledLink` vs `'a'`. Just did a double check of the component

Reply:
Alright I'll correct it